ci/consumer-smoke/Makefile

```
@@ -0,0 +1,57 @@
+	-I$(LWIP_PATH)/src/include \
+	-I$(FREERTOS_KERNEL_PATH)/include \
+	-I$(FREERTOS_KERNEL_PATH)/portable/GCC/ARM_CM3
+
+CFLAGS := -mcpu=cortex-m3 -mthumb -Wall -Wextra -Wpedantic -Werror \
+	$(SOLIDSYSLOG_INCLUDES) $(UPSTREAM_INCLUDES)
+
+OBJS := $(patsubst $(SOLIDSYSLOG_DIR)/%.c,$(BUILD)/%.o,$(SOLIDSYSLOG_SRCS)) \
+	$(BUILD)/main.o
+
+.PHONY: all clean
+all: $(BUILD)/libconsumer_smoke.a
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)